### PR TITLE
BOM wave archive - fix processing of text file

### PR DESCRIPTION
**Pull request by lbesnard**
should fix https://github.com/aodn/content/issues/445

### CODE REVIEW

**Comment by lbesnard** [comment]
will need to check new outputs:
[bom.tar.gz](https://github.com/aodn/data-services/files/3834705/bom.tar.gz)

and also remove bad xlsx from archive dir

**Comment by bpasquer** [comment]
Note that Cape Du Couedic data 2000-2007 also need reprocessing

**Comment by lbesnard** [comment]
@bpasquer files are on https://cloudstor.aarnet.edu.au/plus/s/Hzrk5EUKDNezvfp

File: AODN/AODN-WAVE-DM/BOM-WAVE-DM/bom_wave_library/bom_wave_parser.py [hunk 1]
```diff
@@ -120,6 +120,41 @@ def parse_csv_bom_wave(filepath):
...
+def parse_txt_bom_wave(filepath):
+    """
+    parser for csv bom wave files
+    :param filepath:
+    :return: dataframe of data
+    """
+    if filepath.endswith('.txt'):
+        col_lengths = {'datetime': range(1, 20),
+                       'Hs': range(20, 25),
+                       'Hrms': range(20, 25),
```
Comment (bpasquer) [inline AODN/AODN-WAVE-DM/BOM-WAVE-DM/bom_wave_library/bom_wave_parser.py]: You duplicated the column range for Hs and Hrms

**Pull request merged**